src/hooks/useIsMobileView.ts

```diff
@@ -5,6 +5,7 @@ export default function useIsMobileView() {
 	const isMobile = useMediaQuery({ maxWidth: 600 });
 	const isTablet = useMediaQuery({ minWidth: 600, maxWidth: 1_023 });
 	const isDesktop = !isMobile && !isTablet;
+	const isLessthan768 = useMediaQuery({ maxWidth: 768 });
 
-	return { isMobile, isTablet, isDesktop };
+	return { isMobile, isTablet, isDesktop, isLessthan768 };
```

Comment:
If you need a special case, use it directly where its needed. It better that way.